testplan/web_ui/testing/src/Nav/InteractiveNavList.js

```diff
@@ -14,7 +15,7 @@ const InteractiveNavList = (props) => {
     props,
     (entry) => (
       <InteractiveNavEntry
-        name={entry.name}
+        name={_.isEmpty(entry.part) ? entry.name : entry.uid}
```

Comment:
I thought this is before reset_uid() so we could always use entry.uid, no?

Reply:
seems we can use entry.uid